test/lib/code_corps/github/oauth_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule CodeCorps.GitHub.OAuthTest do
+  use CodeCorps.ModelCase
+
+  alias CodeCorps.{
+    GitHub, User
+  }
+
+  alias Ecto.Changeset
+
+  setup do
+    bypass = Bypass.open
+    GitHub.Bypass.setup(bypass)
+
+    on_exit fn ->
+      GitHub.Bypass.teardown()
+    end
+
+    {:ok, bypass: bypass}
+  end
```

Comment:
Not sure if we could have a different case for this, or some other way to use more quickly this `setup` in tests in the future without copying/pasting everywhere.